mets_mods2tei/api/mets.py

```diff
@@ -514,6 +517,12 @@ def get_alto(self, phys_id):
 
     def get_order(self, phys_id):
         """
-        Return the manually set order for a given physical ID
+        Return the logical (manually set) page number for a given physical ID
         """
-        return self.order_map.get(phys_id, "-1")
+        return self.order_map.get(phys_id, "0")
```

Comment:
Uh, oh. I assume you have checked why I decided to use `-1` in the first place?

Reply:
Yes, because of sorting . But you achieve the same with zero. And the change is needed to make the final TEI `pb/@facs` (or rather, `pb/@n`) a valid page number. (It used to be `"-0001"` everywhere.)